test/fixtures/ecdsa.json

```diff
@@ -191,8 +170,8 @@
         "description": "Invalid r value (nR is infinity)",
         "exception": "nR is not a valid curve point",
         "e": "01",
-        "signature": {
-          "r": "fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141",
+        "signatureRaw": {
+          "r": "fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364140",
```

Comment:
And these were hex, but being interpreted as decimal.
Now they are all hex, and the test fixtures amended.